src/credentials/sso/ssoSupport.ts

```diff
@@ -32,3 +34,17 @@ export function isSsoProfile(profile: Profile): boolean {
     }
     return false
 }
+
+export async function displaySsoProfileChosenMessage(): Promise<void> {
+    const signInInstructionMessage = `You have chosen an AWS Single Sign-On profile that requires authorization. `
+    await vscode.window.showInformationMessage(signInInstructionMessage, { modal: true })
+}
+
+export async function openSsoPortalLink(authorization: StartDeviceAuthorizationResponse): Promise<boolean> {
```

Comment:
Does this need its own function? Can you just `return await vscode.env.openExternal(vscode.Uri.parse(authorization.verificationUriComplete!))` ?